server/server.go

```diff
@@ -489,7 +489,7 @@ func (s *Server) startServer(ctx context.Context) error {
 	s.safePointV2Manager = gc.NewSafePointManagerV2(s.ctx, s.storage, s.storage, s.storage)
 	s.hbStreams = hbstream.NewHeartbeatStreams(ctx, s.clusterID, "", s.cluster)
 	// initial hot_region_storage in here.
-	if !s.IsAPIServiceMode() {
+	if !s.IsServiceEnabled(mcs.SchedulingServiceName) {
```

Comment:
When running `startServer`, is the `RaftCluter` running?

Reply:
Yes